common/network_manager.go

```diff
@@ -347,37 +347,52 @@ func (m *NetworkManager) onLoadingFailed(event *network.EventLoadingFailed) {
 }
 
 func (m *NetworkManager) onLoadingFinished(event *network.EventLoadingFinished) {
-	req := m.requestFromID(event.RequestID)
+	rid := event.RequestID
```

Comment:
s/rid/reqID ?

Reply:
Sorry, too long :) It feels more idiomatic to use `rid`. Thanks for the suggestion, though.